libraries/cli/include/cli/testnet_config.hpp

```diff
@@ -102,6 +102,7 @@ const char *testnet_json = R"foo({
           "deposit_delay": "0x5",
           "withdrawal_delay": "0x5",
           "eligibility_balance_threshold": "0xf4240",
+          "coins_per_vote": "0xf4240",
```

Comment:
why is this called coins_per_vote and not eligibility_balance_step ? Would that be better ?

Reply:
Maybe `vote_eligibility_balance_step` is better?

Reply:
yeah even better, I skipped "vote_" so it has the smae form as eligibility_balance_threshold, which is missing vote_ prefix too